common/src/DefaultWebProxySettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Shared;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Client.Common
```

Comment:
I don't think we need another common namespace but if you think we do, it should be outside of Client (it's used everywhere).
Logging is using `Shared`.